### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'values')

### ISSUE THREAD

**Issue by nbdn** [ssut/nestjs-sqs#53]
Since NestJS released their version 10, the `nestjs-sqs` lib doesn't work anymore, giving the following error message:

```javascript
TypeError: Cannot read properties of undefined (reading 'values')
```

Regarding the stack trace, I can see that it comes from the `@nestjs-plus/discovery` lib used internally, which is a fork and hasn't been updated since 4 years...

You can find a reproductible repository example [here](https://github.com/nbdn/nestjs-10-sqs-issue), just start the app in development mode and you will see the error message.

**Comment by DennisSnijder** [comment]
@ssut mind creating a new release with the fix? 👼

**Comment by nbdn** [comment]
@ssut please tag a new release, we need it to be able to bump to NestJS v10

**Comment by ssut** [comment]
@DennisSnijder @nbdn Sorry for the delay. I just have published [v2.0.1](https://www.npmjs.com/package/@ssut/nestjs-sqs/v/2.0.1), which includes this fix. Please check it.
